Lamina/include/Math/timer.hpp

```diff
@@ -0,0 +1,52 @@
+#ifndef LM_TIMER
+#define LM_TIMER
+
+#include <chrono>
+#include <iostream>
+
+#define LM_NANOSECONDS		0b00001
```

Comment:
This is a very archaic way of doing constants. It works fine but generally you want to favor using normal constant variables or in this case, an enum might be better.